core/evancore.vue.libs/src/components/profile/profile-picture/profile-picture.scss

```diff
@@ -115,4 +115,7 @@
   &.sm {
     @include masks(30px);
   }
+  &.sm-plus {
+    @include masks(56px);
```

Comment:
can you check with design if we really need this? I think 3 sizes should be actually enough, maybe we can kick out some of the others and utilize it for this.

Reply:
As you can see the differences in dimensions are pretty big. I don't see how we would not need this? Don't think refactoring the whole component, and all places that use it, is in the scope of this story :)